saleor/core/utils/random_data.py

```diff
@@ -203,9 +204,17 @@ def create_product_variants(variants_data):
             continue
         defaults['product_id'] = product_id
         defaults['attributes'] = json.loads(defaults['attributes'])
+        reset_currency(defaults, 'price_override', settings.DEFAULT_CURRENCY)
```

Comment:
I think I'd opt for more explicit version, like:
```
defaults['price_override'] = get_in_default_currency(
    defaults['price_override'])
```
I don't like the idea of modifying the dictionary under the hood

Reply:
YEa, I had the same doubt at the time, will implement suggested changes later.

Reply:
should be fixed now.